src/covid_shared/cli_tools/decorators.py

```diff
@@ -169,3 +169,11 @@ def add_snapshot_dependency_option():
     Declares a dependency on a specific snapshot version. Defaults to "best".
     """
     return _create_dependency_option('snapshot')
+
+
+def validate_options_with_q(quick: int, mark_best: bool, production_tag: str):
+    """
+    Quick runs should never be marked 'best' or for production
+    """
+    if (bool(quick)) and (mark_best or bool(production_tag)):
```

Comment:
You don't need the explicity `bool` calls here.  Python calls `__bool__` on objects in a setting where truth values are required on it's own.